qiskit/circuit/controlledgate.py

```diff
@@ -95,12 +97,14 @@ def __init__(
            qc2.draw('mpl')
         """
         self.base_gate = None if base_gate is None else base_gate.copy()
-        super().__init__(name, num_qubits, params, label=label)
-        self._num_ctrl_qubits = 1
-        self.num_ctrl_qubits = num_ctrl_qubits
+        super().__init__(name, num_qubits, params, label=label, duration=duration, unit=unit)
+        if not hasattr(self, "_num_ctrl_qubits"):
```

Comment:
I removed all of this logic because it's not needed anymore in the current form of this PR: https://github.com/Qiskit/qiskit/pull/10898/commits/3a418902678dea4168b8083b4fee891102e206f6